packages/p/perl-pcsc/monitoring.yml

```diff
@@ -0,0 +1,6 @@
+releases:
+  id: 2613
+  rss: ~
```

Comment:
Shouldn't this be the version check URL from release-monitoring.org?
[LudovicRousseau/pcsc-perl/tags](https://github.com/LudovicRousseau/pcsc-perl/tags)

Reply:
Yes, I will add it. Most of the perl module only lives in CPAN, so I did not bother searching their homepages. Thanks